### ISSUE TITLE
Introduce default value for "bintray.pkg.name"

### ISSUE THREAD

**Issue by szczepiq** [mockito/shipkit#344]
One of the principles of Shipkit is ease of use. We achieve it by providing sensible defaults for every possible configuration option.

Let's add default value for "bintray.pkg.name" setting, we can use "project.getGroup()" for this. Here's how currently we set the default for Bintray extension in code: [code link](https://github.com/mockito/shipkit/blob/master/src/main/groovy/org/shipkit/internal/gradle/bintray/ShipkitBintrayPlugin.java#L74).

For testing, we can remove the "bintray.pkg.name" setting from Shipkit example project and start relying on the defaults: [code link](https://github.com/mockito/shipkit-example/blob/master/gradle/shipkit.gradle#L27).

Pardon me if the code links get out of date :) Let us know if this ticket is unclear!

**Comment by ArturSkowronski** [comment]
Hey @szczepiq I would like to take this one!

**Comment by szczepiq** [comment]
Cool, here's the place in code where we currently need to specify the name of the package: [link](https://github.com/mockito/shipkit-example/blob/master/gradle/shipkit.gradle#L17). It looks like this:

```Gradle
allprojects {
    plugins.withId("org.shipkit.bintray") {
        bintray {
            pkg {
                repo = 'examples'
                user = 'szczepiq'
                userOrg = 'shipkit'                
                name = "basic"  //  <---***---- H E R E!
                licenses = ['MIT']
                labels = ['continuous delivery', 'release automation', 'mockito', 'shipkit']
            }
        }
    }
}
```

Let's make Shipkit assign sensible default for this setting, so that users don't have to configure it. Convention over configuration!

Let us know if you have further questions :)

**Comment by wwilk** [comment]
Fixed by #355
